packages/help-center/src/data/types.ts

```diff
@@ -1,3 +1,5 @@
+import type { SiteDetails, HelpCenterSite } from '@automattic/data-stores';
```

Comment:
I wonder if we could just merge `./data/types.ts` with `./types`. With the move it no longer makes sense to have 2 separate files for those types IMHO.